packages/js/product-editor/src/hooks/use-feedback-bar/use-feedback-bar.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { resolveSelect, useDispatch, useSelect } from '@wordpress/data';
+import {
+	ALLOW_TRACKING_OPTION_NAME,
+	useCustomerEffortScoreModal,
+} from '@woocommerce/customer-effort-score';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PRODUCT_EDITOR_SHOW_FEEDBACK_BAR_OPTION_NAME,
+	PRODUCT_EDITOR_FEEDBACK_CES_ACTION,
+} from '../../constants';
+
+export const useFeedbackBar = () => {
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+
+	const { wasPreviouslyShown, isLoading: isCesModalOptionsLoading } =
+		useCustomerEffortScoreModal();
+
+	const { shouldShowFeedbackBar } = useSelect(
+		( select ) => {
+			const { getOption, hasFinishedResolution } =
+				select( OPTIONS_STORE_NAME );
+
+			const showFeedbackBarOption = getOption(
+				PRODUCT_EDITOR_SHOW_FEEDBACK_BAR_OPTION_NAME
+			) as string;
+
+			const allowTrackingOption =
```

Comment:
Ah, cool.... I'll make that change. So, in general, should we never have to get the tracking option client-side by fetching it?

Reply:
Exactly.  The only case where it could become out of sync is when it gets enabled client-side without a refresh, but that won't happen if anyone doing this is hitting the correct tracks API (`window.wcTracks.enable`) to enable this:

https://github.com/woocommerce/woocommerce/blob/607324d90f7281617bb7bcf1e385760cd93a99e1/plugins/woocommerce/includes/tracks/class-wc-site-tracking.php#L156